nixos/modules/system/boot/plymouth.nix

```diff
@@ -38,6 +38,14 @@ in
 
       enable = mkEnableOption "Plymouth boot splash screen";
 
+      font = mkOption {
+        default = "${pkgs.dejavu_fonts.minimal}/share/fonts/truetype/DejaVuSans.ttf";
```

Comment:
it would be nice to be able to figure what font is the default sans font for a system. this is just a comment and not a suggestion, but I feel it would be pretty boring to do this with `passthru` listing all the paths to a font within a package, and making a `fonts.fonts.primaryFont`, and fonts having `$variant.path` attr.

Reply:
The relevant font here is not a system default, but one [specified by the theme](https://github.com/KDE/breeze-plymouth/blob/v5.20.5/breeze/breeze.script.cmake#L20-L21).

Finding the default sans font is a simple matter of running `fc-match -f '%{file}' Sans` in an environment with the appropriate Fontconfig configuration. Or if we could somehow pull out the value `Noto Sans` from the selected Plymouth theme, we could run `fc-match -f '%{file}' 'Noto Sans'`. Unfortunately Plymouth themes are written in a custom scripting language and I don’t see a way to run them except in Plymouth itself. We could maybe write some atrocious regex that works most of the time.